internal/pkg/composable/providers/kubernetes/README.md

```diff
@@ -1,20 +1,32 @@
-##  Autodiscover provider in Elastic Agent
+## Autodiscover provider in Elastic Agent
 
-https://www.elastic.co/guide/en/fleet/8.0/dynamic-input-configuration.html#dynamic-providers
+  
 
-Currently Kubernetes dynamic provider can only be configured in [standalone](https://github.com/elastic/beats/blob/03bf16907bea9768427f8305a5c345368b55d834/deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml#L24) agent.
-In fleet managed agent it is enabled by default with default values.
+https://www.elastic.co/guide/en/fleet/current/kubernetes-provider.html
 
-Template based autodiscover of Kubernetes resources is only supported in standalone mode as of now.
-It is not part of the Kubernetes Integration yet.
+  
 
-Hints based autodiscover is not supported yet.
+Kubernetes provider can be configured both in **standalone** and **fleet managed** elastic agent.
 
-### Template based autodiscover
+a. In  **standalone** elastic agent, user needs to edit the [elastic-agent-standalone-kubernetes.yaml](https://github.com/elastic/elastic-agent/blob/f994f5bfdf68db27902a4175c3b655b4d611cf7c/deploy/kubernetes/elastic-agent-standalone-kubernetes.yaml#L28)
+b. In **fleet managed** elastic agent, kubernetes provider is enabled by default with default values. User needs to follow the steps described in the [documentation](https://www.elastic.co/guide/en/fleet/current/advanced-kubernetes-managed-by-fleet.html) to configure it.
```

Comment:
I feel like a blank line is missing from here

Reply:
Does that cause it to render different? I don't think so.

Reply:
it actually does, now **a.** and **b.** are in the same paragraph which makes it kind hard to distinguish the two. However if you add an empty line between **a.** and **b.** they appear properly separated 🙂

Reply:
Then absolutely fix before merge.

Reply:
A blank line is needed indeed